Makefile.include

```diff
@@ -521,3 +521,19 @@ endif
 CFLAGS := $(patsubst -D%,,$(CFLAGS))
 CFLAGS := $(patsubst -U%,,$(CFLAGS))
 CFLAGS += -include '$(RIOTBUILD_CONFIG_HEADER_C)'
+
+# Set a custom channel if needed
+ifneq (,$(filter netdev2_ieee802154,$(USEMODULE)))
```

Comment:
It is really hard to get to a common system for different radio standards. Not only uses every standard its own numerations for channels, but also think about channel hopping etc. So I wouldn't even dare to go there. For each radio standard (or at least the only one we support fully right now: IEEE 802.15.4) there is a central macro e.g. `IEEE802154_DEFAULT_CHANNEL` to set that.